arch/arm/soc/st_stm32/stm32f2/Kconfig.defconfig.stm32f205xx

```diff
@@ -0,0 +1,18 @@
+# Kconfig - ST Microelectronics stm32f205xx MCU
+#
+# Copyright (c) 2018 qianfan Zhao <qianfanguijin@163.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32F205XB || SOC_STM32F205XC || SOC_STM32F205XE || SOC_STM32F205XF || SOC_STM32F205XG
```

Comment:
Please focus on the SoC you're introducing.

Reply:
sorry I don't know waht is foucus, can you explain more clearer? seems that if I use nucleo-f207, I need remove  all soc except f207?

Reply:
I meant "limit to", so yes, if you target nucleo_f207ZG, please keep only f207xx